src/main/java/org/myrobotlab/service/meta/RuntimeMeta.java

```diff
@@ -49,6 +49,12 @@ public RuntimeMeta() {
     // ws client sockets
     addDependency("org.asynchttpclient", "async-http-client", "2.12.3");
 
+    // Nullability annotations
+    // Contrary to what its group ID implies, this is
+    // the reference implementation for javax.annotation.Nullable / NotNull
+    // Apache 2.0 license
+    addDependency("com.google.code.findbugs", "jsr305", "3.0.2");
+
```

Comment:
should it be updated to spotbugs ?

Reply:
That depends, the FindBugs version adheres to jsr305 while spot bugs does not, but the spot bugs version is actively developed and comes with additional useful annotations

Reply:
what do you think ?

Reply:
I've looked through a few different options, and it seems that the most widely-used nullability annotations are from the Checker Framework, which also includes some pretty nifty annotations to help with logic bugs. I think that's probably the best option I can see at the moment, it seems a lot of large projects like Guava use it. I'm not sure integrating it in this PR is a good idea though, so I'll probably remove JSR305 and the annotations for now and make a new PR for Checker later

Reply:
JSR305 removed and the annotations commented out (so I can convert them to Checker later), I'll make a new PR to add Checker stuff later